intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -399,13 +402,13 @@ To use this dataset, we will load it from the `palmerpenguins` package using a `
 penguins <- palmerpenguins::penguins
 ```
 
-![drawings of penguin species](diagrams/lter_penguins.png) Artwork by @allison_horst
+![drawings of penguin species](diagrams/lter_penguins.png) Artwork by [@allison_horst](https://twitter.com/allison_horst)
```

Comment:
Noting she specifically wants to be tagged at @allison_horst, but her github is sans underscore. So, we'll link to her twitter which is this handle.